speech/unit_tests/test_client.py

```diff
@@ -91,18 +100,19 @@ def test_sync_recognize_content_with_optional_parameters(self):
             }
         }
         credentials = _Credentials()
-        client = self._makeOne(credentials=credentials)
+        client = self._makeOne(credentials=credentials, use_gax=False)
         client.connection = _Connection(RETURNED)
```

Comment:
Isn't `connection` an argument for the constructor? PS Sorry for just noticing, saw it because the line above it got edited.